Source/Scene/Expression.js

```diff
@@ -343,6 +369,34 @@ define([
         return feature.getProperty(this._value);
     };
 
+    function checkFeature (ast) {
+        return (ast._value === 'feature');
+    }
+
+    // PERFORMANCE_IDEA: Determine if parent property needs to be computed before runtime
```

Comment:
For our spec, should the expression in brackets be required to be a literal?  If so, this optimization should be natural and I suspect we could do it now.  If not, I guess we need tests for cases like `${foo[${memberName}]}`, which is overkill for now IMO if it is painful.